REQUIREMENTS.html

```diff
@@ -214,6 +216,7 @@ <h3>Optional packages:</h3>
  
 <li><b>Subversion</b> (svn in g.extension to fetch code selectively from grass-addons on GitHub)<br>
 <a href="https://subversion.apache.org/">https://subversion.apache.org/</a><br>
+</li>
```

Comment:
```suggestion
<a href="https://subversion.apache.org/">https://subversion.apache.org/</a>
</li>
```

Reply:
Why not?

Reply:
But it's really not the only one in the file. Would you want a more complete review of the file?

Reply:
I wonder if .html is the right format here while INSTALL.md/CONTRIBUTING.md etc are meanwhile markdown files, more appropriate for git.

Reply:
I couldn't figure out the effects of this, like what links to this in all the sources and wikis and all, (or what copies it/packages it), so the easiest for now was to simply remove some easy linting errors. That's a discussion for a backlog item perhaps?

Reply:
If I may, I think md format would be easier to mantain and more consistent with install and contributing files. @echoix would you be willing to convert it?

Reply:
If you can find the references elsewhere in the wiki or anything else, no problem, I think I can try :)

Reply:
I used something equivalent, it's quite easy, I'm just proofreading everything